common/script/content/index.js

```diff
@@ -1251,7 +1251,7 @@ api.quests = {
       str: 1.5
     },
     drop: {
-      items: [
+      itemsForOwner: [
```

Comment:
Yeah having `item.onlyOwner` or something similar seems the best solution. These are small arrays and it's not going to be a perf problem having to filter them